cmd/kubeadm/app/cmd/reset.go

```diff
@@ -69,6 +69,9 @@ type resetOptions struct {
 	cleanupTmpDir         bool
 }
 
+// compile-time assert that the local data object satisfies the phases data interface.
+var _ phases.ResetData = &resetData{}
```

Comment:
this doesn't seem right.
compile time assertion can be done in just one place data_test.go which means we don't have to make resetData public

Reply:
ok, thank you for your review~